src/SubModules/ShellModule.jl

```diff
@@ -130,4 +132,72 @@ end
 computes the number of basisfunctions (cartesian) described by the shell.
 """
 nbf(shell::Shell) = BaseModule.numberMQNsCartesian(shell.lqn)
+
+"""
+    computePattern(basis)
+returns the matrix blocking pattern for a given basis
+"""
+function computePattern(basis::Array{ShellModule.Shell,1})
```

Comment:
basis is a bit misleading for the argument name, because it isn't a Basis (e.g. `GaussianBasis`). How about `shells::Vector{Shell}`?

Reply:
Good point I changed all instances of ``` basis ``` to ``` shells ```.